server.js

```diff
@@ -1,5 +1,28 @@
-const app = require('./app')
+require("dotenv").config();
+const PORT = process.env.PORT || 3000;
+const uriDb = process.env.DB_HOST;
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+const mongoose = require("mongoose");
+const app = require("./app");
+
+const connection = mongoose.connect(uriDb, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
```

Comment:
Sprawdzałeś czy to jest jeszcze potrzebne?

Reply:
Pamiętam że w starej wersji tak ale że w nowej connect  już tego nie oczekuje. Zobacz i się upewnij, bo po co trzymać coś co nie działa ;)